go/codegen/experimental/codegen_normal_stmt.go

```diff
@@ -24,11 +24,16 @@ var normalStmtStepTmpl = `
 def step_entry_{{.StepIndex}}():
     import runtime
     import runtime.dbapi
+    from runtime.dbapi import table_writer
+
     conn = runtime.dbapi.connect("{{.DataSource}}")
     stmt = """{{.Stmt}}"""
     if conn.is_query(stmt):
```

Comment:
FYI conn.is_query is not implemented yet

Reply:
`conn.is_query` is already there at develop branch.